VERSION

```diff
@@ -1,2 +1,2 @@
-WAZUH-ANSIBLE_VERSION="v4"
-REVISION="40000"
+WAZUH-ANSIBLE_VERSION="v4.3"
+REVISION="40215"
```

Comment:
This should be v4.3.0 and revision 40301 -> https://github.com/wazuh/wazuh/blob/master/src/REVISION

Reply:
Done!